nselib/data/http-default-accounts-fingerprints.lua

```diff
@@ -1675,6 +1675,40 @@ table.insert(fingerprints, {
   end
 })
 
+table.insert(fingerprints, {
+  name = "Dell iDRAC9",
+  cpe = "cpe:/o:dell:idrac9_firmware",
+  category = "console",
+  paths = {
+    {path = "/"}
+  },
+  target_check = function (host, port, path, response)
+    -- analyze response for 1st request to "/"
+    if not (response.status == 302 and (response.header["location"] or ""):find("/restgui/start.html$")) then return false end
+
+    -- check with 2nd request to "/restgui/start.html" to be sure
+    local resp = http_get_simple(host, port, url.absolute(path, "/restgui/start.html"))  
```

Comment:
Calling `url.absolute()` with an absolute path as the second argument defeats the purpose of the call. The result will be merely equal to this argument, not its logical concatenation with `path`, which is what you want here.

Reply:
Indeed thx!